bootstrap/bootstrap.go

```diff
@@ -35,6 +37,23 @@ func Start(ctx context.Context, cfg *config.Config) error {
 		return err
 	}
 
+	reg := prometheus.NewRegistry()
+	factory := promauto.With(reg)
+	options := metrics.StorageSizeCollectorOpts{
+		Factory:    factory,
+		Logger:     logger,
+		StorageDir: cfg.DatabaseDir,
+		Interval:   cfg.StorageSizeUpdateInterval,
+	}
```

Comment:
this as well, you can just pass in the required parameters from the config, or even whole config, but don't expose configuration in the bootstrap.